src/Settings/Sidebar.jsx

```diff
@@ -80,7 +89,7 @@ return (
   <Wrapper>
     <Title>Settings</Title>
 
-    {filteredMenuItems.map((item) => (
+    {filteredMenuItems.length > 1 && filteredMenuItems.map((item) => (
```

Comment:
@shelegdmitriy why only return a menuItem if there is > 1?

Reply:
Ah, because it defaults to iDOS?

Reply:
Correct. At this moment we have only one tab so it doesn't make much sense to show it